tests/common_test.go

```diff
@@ -85,22 +85,39 @@ func (s *QemuSuite) runQemu(args ...string) error {
 func (s *QemuSuite) WaitForSSH() error {
 	sshArgs := []string{
 		"--qemu",
+		"true",
+	}
+
+	var err error
+	for i := 0; i < 100; i++ {
```

Comment:
50 seconds? if that's how long it takes - wow :)

mind you, later we should parse the boot output and "know" if it ran as expected.